resources/js/filters/toArrayFilter.js

```diff
@@ -0,0 +1,9 @@
+app.filter("toArray", function(){
```

Comment:
I think we can have all the filters in a single file, will decrease the num of files being loaded